colmi_r02_client/real_time/packet.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import dataclass
+
+from colmi_r02_client.real_time.enum import RealTimeReading, Action
+from colmi_r02_client.packet import make_packet
+
+CMD_START_REAL_TIME = 105
+CMD_STOP_REAL_TIME = 106
+
+@dataclass
+class Reading:
+    kind: RealTimeReading
+    value: int
+
+@dataclass
+class ReadingError:
+    kind: RealTimeReading
+    code: int
```

Comment:
These dataclasses can be `(frozen=True)`?

Reply:
makes sense! i didn't know you could do that